src/Tests/dotnet-workload-install.Tests/GivenDotnetWorkloadInstall.cs

```diff
@@ -165,10 +165,10 @@ public void GivenNoWorkloadsInstalledInfoOptionRemarksOnThat()
             // However, we can test a setup where no workloads are installed and --info is provided. 
 
             _reporter.Clear();
-            var mockWorkloadIds = new WorkloadId[] { new WorkloadId("xamarin-android"), new WorkloadId("xamarin-android-build") };
+            _ = new WorkloadId[] { new WorkloadId("xamarin-android"), new WorkloadId("xamarin-android-build") };
```

Comment:
Why do you need to make these at all if they're just throwaways?

Reply:
That is certainly a question... 🤔 Let me dig in the history. If they're remnants from the past, I'll just remove them.

Reply:
After some sleuthing, it was @nagilson in this commit! Dun dun daaaaaaaaa. 🙈
https://github.com/dotnet/sdk/commit/011e1e8b86cc4f2772db191a6373b4c8bd5d225d#diff-c8bc89094b206b506f35f0cb9ac6e55591ae2468130efe0a19dab1810c0f0c1cL185-R191

Looks like an artifact of the previous implementation of the test. I'll remove these lines.